missing log lines due to terminal escape sequences (when running with --verbose)

OriHoch opened the pull request:
Couldn't figure out how to test it automatically but I added a test which will hopefully show the problem on travis

akariv commented:
This actually makes sense.
Solution would be to remove the nifty progress display when running in verbose mode.

OriHoch commented:
@akariv thanks, fixed in the cli, updated the test to ensure there are no escape sequences when running with --verbose (and that there are when running without verbose)
